### ISSUE TITLE
Include Consul `ServiceName` field in `consul_health_service_status` metric labels (as before)

### ISSUE THREAD

**Issue by clarenceb** [prometheus/consul_exporter#15]
A recent change made in commit "Fix healthy metric and remove redundant total metric" (4ba4dcaa257575ba0cfb74ed53d2a1a3c49c4d51) changed the value of label "service" from entry.Service.Service (Consul ServiceName) to entry.Service.ID (Consul ServiceID).

This change makes it more difficult for me to easily aggregate over related service instances.  E.g. how many instances are healthy for a particular service?

E.g. Before I used to be able to do:

**Total healthy instance count per node**
`sum by (node, service) (consul_health_service_status{service="myservicename"})`
or
**Total healthy instance for cluster**
`sum by (service) (consul_health_service_status{service="myservicename"})`

This won't work anymore.

My proposal is to have a method for the user to choose additional Consul service fields to include in the `consul_health_service_status` metric, like `ServiceName` (mainly this one) and `ServiceTags`.  Then users can decide which fields are important to them.

Or a simpler solution is to just include `ServiceName` in addition to `ServiceID` and not have customisation.

I am happy to submit a pull request if this is something you think is of use to others.  As I need this solution asap for a workshop I am running I will "hack" something for myself now based on the simpler solution.

**Comment by guoshimin** [comment]
+1

**Comment by grobie** [comment]
The previous implementation only exported the information of a single instance per node, even if multiple were running on it. Only the service ID is unique per node and has to be included in the exported labels.

@clarenceb Can you give me an example of your serviceIDs and ServiceNames? We set our ServiceIDs explicitly and can do something like that:

```
sum by (node, service) (consul_health_service_status{service=~"myservicename.*"})
```

**Comment by nickvth** [comment]
+1

**Comment by NoumanSaleem** [comment]
+1 for this. Can't combine a service over multiple nodes since `service` is unique

```
sum by (node,service)(consul_health_service_status{service=~".*foobar.*"})
Element	Value
{node="i-xxx",service="95a25836c5c7:service-foobar-89-foobar-868ad9aed089e0895400:80"}	1
{node="i-xxx",service="efa3a11ecc10:service-foobar-90-foobar-fcf198b4e6f3f1c3ef01:80"}	1
```

**Comment by jeinwag** [comment]
@grobie We're using registrator with consul, and registrator sets the service ID based on host name, container name and (internal) port. The service name itself isn't necessarily part of the service ID. It would be very convenient if consul_exporter also exposed the service name, e.g. for checks on how many healthy instances of a certain service are there.

**Comment by awwithro** [comment]
+!

**Comment by grobie** [comment]
This has been implemented in #25.
